storage/tianmu/core/aggregation_algorithm.cpp

```diff
@@ -201,7 +201,7 @@ void AggregationAlgorithm::Aggregate(bool just_distinct, int64_t &limit, int64_t
     }
   } else {
     int64_t local_limit = limit == -1 ? upper_approx_of_groups : limit;
-    MultiDimensionalGroupByScan(gbw, local_limit, offset, sender, limit_less_than_no_groups, false);
+    MultiDimensionalGroupByScan(gbw, local_limit, offset, sender, limit_less_than_no_groups);
```

Comment:
I dont think this implementation is a good one. usually, we add some new query plan to generate parallel operators. and these parallel operator do real job.  in this way, we can know exactly how stonedb executes a big task in parallel.

and It's hard to find out why the performace of parallel aggregation does not meet the expectation.

Reply:
For tianmu, there is a general lack of output for all execution plans, plus query plans such as explain, but in terms of development planning, new iterations are needed to launch such a large requirement